docs/deployment-with-docker.md

```diff
@@ -0,0 +1,67 @@
+# Deployment with Docker
+A Docker image is provided in order to run WDK from scratch with a single command without requiring tooling or a java project.
+
+Symphony WDK released docker images are published as a [Github package](https://github.com/orgs/finos/packages?repo_name=symphony-wdk).
+
+### Pull Docker image
+Pull the docker image. You can replace the tag "latest" with the [release tag](https://github.com/finos/symphony-wdk/tags) you want.
+```shell
+ $ docker pull ghcr.io/finos/symphony-wdk:latest
+```
+
+### Mount a shared folder
+Mount a folder on your machine to be shared with the Docker container. The shared folder should contain:
```

Comment:
should we precise the mounted folder's name ? as the `ENTRYPOINT` in dockerfile uses `./data`, in addition in the doc below, it mentions that the keys and /workflows folder are under `./symphony` folder, is it another volume being mounted or should it be aligned with `./data` folder ?

Reply:
my bad, you're right. I think 'symphony' sounds better than 'data'. I updated dockerfile accordingly.

Reply:
Well actually we can make it an env variable, with "symphony" as default value. WDYT? let me do that in next commit.

Reply:
think it's not very important to have the WORKDIR customisable as long as people know which path to mount their config - we can leave it static in the Dockerfile and make the documentation easier